testsuite/tests/lib-unix/unix-execvpe/exec.ml

```diff
@@ -1,3 +1,11 @@
+(* TEST
+   include unix
+   script = "${test_source_directory}/has-execvpe"
+   * script
+   ** bytecode
+   ** native
```

Comment:
This is a tree of tests. `script` runs first, and its children `bytecode` and `native` are run if it passes. Note that the magical return code 125 means to skip, not fail.